trace-sdk/src/main/java/io/bitrise/trace/data/collector/view/ApplicationStartUpDataListener.java

```diff
@@ -132,12 +138,8 @@ public void onActivityResumed(@NonNull final Activity activity) {
      * @see
      * <a href="https://developer.android.com/topic/performance/vitals/launch-time">https://developer.android.com/topic/performance/vitals/launch-time</a>
      */
-    @Nullable
+    @NonNull
     private synchronized ApplicationStartData getApplicationStartData() {
-        if (!isApplicationStart) {
```

Comment:
removed these lines of code because this method is private, and only one place that calls it - and that is surrounded by the check already - so this check was unnecessary.